fpdf/table.py

```diff
@@ -250,17 +250,18 @@ def render(self):
             pagebreak_height = rows_info[i].pagebreak_height
             # pylint: disable=protected-access
             page_break = self._fpdf._perform_page_break_if_need_be(pagebreak_height)
-            if (
-                page_break
-                and self._fpdf.y + pagebreak_height > self._fpdf.page_break_trigger
-            ):
-                # Restoring original position on page:
-                self._fpdf.x = prev_x
-                self._fpdf.y = prev_y
-                self._fpdf.l_margin = prev_l_margin
-                raise ValueError(
-                    f"The row with index {i} is too high and cannot be rendered on a single page"
-                )
+            # Modification pour issue #1460 : On autorise le dépassement de page
+            # if (
+            #     page_break
+            #     and self._fpdf.y + pagebreak_height > self._fpdf.page_break_trigger
+            # ):
+            #     # Restoring original position on page:
+            #     self._fpdf.x = prev_x
+            #     self._fpdf.y = prev_y
+            #     self._fpdf.l_margin = prev_l_margin
+            #     raise ValueError(
+            #         f"The row with index {i} is too high and cannot be rendered on a single page"
+            #     )
```

Comment:
Please delete the commented code block
We don't keep dead code in the repository - git history is good enough if we ever need to recover old code